src/nyc_trees/nyc_trees/settings/production.py

```diff
@@ -41,30 +41,10 @@ def get_env_setting(setting):
 # END HOST CONFIGURATION
 
 # EMAIL CONFIGURATION
-# TODO: We should be using Amazon SES instead
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-backend
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-host
-EMAIL_HOST = environ.get('EMAIL_HOST', 'email-smtp.us-east-1.amazonaws.com')
-
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-host-password
-EMAIL_HOST_PASSWORD = environ.get('EMAIL_HOST_PASSWORD', '')
-
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-host-user
-EMAIL_HOST_USER = environ.get('EMAIL_HOST_USER', 'your_email@example.com')
-
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-port
-EMAIL_PORT = environ.get('EMAIL_PORT', 587)
-
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-subject-prefix
-EMAIL_SUBJECT_PREFIX = '[%s] ' % SITE_NAME
-
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#email-use-tls
-EMAIL_USE_TLS = True
-
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#server-email
-SERVER_EMAIL = EMAIL_HOST_USER
+EMAIL_BACKEND = 'apps.core.mail.backends.boto.EmailBackend'
+EMAIL_AWS_ACCESS_KEY_ID = environ.get('EMAIL_AWS_ACCESS_KEY_ID', None)
+EMAIL_AWS_SECRET_ACCESS_KEY = environ.get('EMAIL_AWS_SECRET_ACCESS_KEY', None)
+EMAIL_AWS_REGION = environ.get('EMAIL_AWS_REGION', None)
```

Comment:
Does it make sense to make the environment variable `AWS_REGION` vs. having it e-mail specific? Regardless, it would probably be safe to default it to `us-east-1`.